lib/gds_api/version.rb

```diff
@@ -1,3 +1,3 @@
 module GdsApi
-  VERSION = '26.4.0'
+  VERSION = '26.4.2'
```

Comment:
Do not increment the version here. Do this in a separate PR, examples: https://github.com/alphagov/gds-api-adapters/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aclosed+bump